test/e2e/operator_test.go

```diff
@@ -151,7 +162,12 @@ func TestAWSLoadBalancerControllerWithDefaultIngressClass(t *testing.T) {
 	t.Log("Creating aws load balancer controller instance with default ingress class")
 
 	name := types.NamespacedName{Name: "cluster", Namespace: "aws-load-balancer-operator"}
-	alb := newALBCBuilder().withName(name).build()
+
+	bldr := newALBCBuilder().withName(name)
+	if isOnROSA() {
+		bldr.withCredSecret(controllerSecretName)
+	}
+	alb := bldr.build()
```

Comment:
Should probably make a helper function for this, it is used 4 other times.

Reply:
Done.